Cython/Includes/cpython/unicode.pxd

```diff
@@ -1,4 +1,9 @@
+from libc.stdint cimport uint8_t, uint16_t
```

Comment:
I don't remember – is `<stdint.h>` safe to include in a platform independent way these days?

Reply:
Hm apparently this is a c99/c++89 feature and only available in msvc since the 2010 version. It appears we already use it in https://github.com/cython/cython/blob/master/Cython/Includes/cpython/time.pxd. I replaced it with
```
ctypedef unsigned char  uint8_t
ctypedef unsigned short uint16_t
```
for now.